client/src/App.js

```diff
@@ -1,11 +1,11 @@
 import "./App.css";
+import React from "react";
+import HomePage from "./component/HomePage";
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <h1>Video Recommendation</h1>
-      </header>
+      <HomePage />
```

Comment:
Very nice! Later on, will we start to use [react-router-dom](https://v5.reactrouter.com/web/guides/quick-start) and keeping your App.js file as clean and simple as possible is very good practice. Nice one Laleh!!!